### DISCUSSION TITLE
Antialiasing solution?

### DISCUSSION THREAD

**Discussion by Martysl1** [fholger/crysis_vrmod#16] [General]
Thank you very much for the awesome Crysis VR mode, well done 😄 . Once again I can reminisce after years of playing Crysis 1.
Will it be possible in the future to make the DLSS/DLAA upscaler work in this mode with TAA to resolve antialiasing? 🤔

**Comment by fholger** [comment]
Not sure, but I wouldn't count on it. To support DLSS, the engine must create a motion vector buffer during rendering as input for DLSS, and this cannot realistically be modded in, it must already exist in the engine. I have not seen any evidence of Crysis generating motion vectors in the frame grabs I've looked at. There is a chance it might do so when you enable motion blur, but since that feature does not work in VR, it is not clear if those could be used in VR even if they were rendered.
